tests/foreman/api/test_capsule.py

```diff
@@ -22,6 +22,7 @@
 @pytest.mark.e2e
 @pytest.mark.upgrade
 @pytest.mark.tier1
+@pytest.mark.pit_server
```

Comment:
PIT currently does not support running capsules. Please remove the marker.

Reply:
removed